mmocr/core/prepare_model.py

```diff
@@ -0,0 +1,120 @@
+import os
+import os.path as osp
+import shutil
+import urllib
+
+
+def prepare_ckpt(checkpoint_file, checkpoint_url):
```

Comment:
These shouldn't exist.
Pls refactor the code as jeffery commented https://github.com/open-mmlab/mmocr/pull/105/files#r617689124